### ISSUE TITLE
"Error: EEXIST, symlink ..." when running electron-rebuild

### ISSUE THREAD

**Issue by jdp** [mafintosh/electron-prebuilt#82]
When running [`electron-rebuild`](https://github.com/paulcbetts/electron-rebuild) to rebuild native modules for use in an electron app, install.js for electron-prebuilt fails with EEXIST.

```
$ ./node_modules/.bin/electron-rebuild
> screen-recorder@1.4.0 install /Users/justinpoliey/Work/streamable-desktop/node_modules/screen-recorder
> node-gyp rebuild

  CXX(target) Release/obj.target/addon/addon.o
  CXX(target) Release/obj.target/addon/ScreenRecorderWrapper.o
  CXX(target) Release/obj.target/addon/ScreenRecorder.o
  SOLINK_MODULE(target) Release/addon.node

> nslog@3.0.0 install /Users/justinpoliey/Work/streamable-desktop/node_modules/electron-rebuild/node_modules/nslog
> node-gyp rebuild

  CXX(target) Release/obj.target/nslog/src/main.o
  CXX(target) Release/obj.target/nslog/src/nslog_mac.o
  SOLINK_MODULE(target) Release/nslog.node

> electron-prebuilt@0.35.4 postinstall /Users/justinpoliey/Work/streamable-desktop/node_modules/electron-prebuilt
> node install.js


/Users/justinpoliey/Work/streamable-desktop/node_modules/electron-prebuilt/install.js:15
  throw err
        ^
Error: EEXIST, symlink '/Users/justinpoliey/Work/streamable-desktop/node_modules/electron-prebuilt/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Electron Framework'
    at Error (native)

npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/Users/justinpoliey/Work/streamable-desktop/node_modules/electron-rebuild/node_modules/npm/bin/npm-cli.js" "rebuild" "--runtime=electron" "--target=0.35.4" "--arch=x64"
npm ERR! node v0.12.7
npm ERR! npm  v2.14.15
npm ERR! code ELIFECYCLE
npm ERR! electron-prebuilt@0.35.4 postinstall: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-prebuilt@0.35.4 postinstall script 'node install.js'.
npm ERR! This is most likely a problem with the electron-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls electron-prebuilt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/justinpoliey/Work/streamable-desktop/node_modules/npm-debug.log
```

**Comment by jdp** [comment]
This happened when I installed the native module after installing `electron-prebuilt`. If I uninstall `electron-prebuilt` and then re-run `electron-rebuild`, it completes without errors.

**Comment by mafintosh** [comment]
this is fixed in 0.36
